integration-test/1491-japanese-shields.py

```diff
@@ -0,0 +1,202 @@
+                'surface': 'paved', 'source': 'openstreetmap.org',
+                'name:en': 'Meiji-dori', 'oneway': 'yes', 'ref': '305',
+                'route': 'road', 'highway': 'primary',
+                'name:ja': u'\u660e\u6cbb\u901a\u308a',
+            }),
+            dsl.relation(1, {
+                'name:en': 'Meiji Dori', 'name': u'\u660e\u6cbb\u901a\u308a',
+                'route': 'road', 'source': 'openstreetmap.org',
+                'wikidata': 'Q8011176', 'type': 'route',
+                'network': 'jp:prefectural',
+            }, ways=[415221603]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 415221603,
+                'shield_text': '305', 'network': 'JP:prefectural',
+            })
+
+    def test_jp_e_expressways(self):
+        import dsl
+
+        z, x, y = (16, 57350, 26039)
+
+        self.generate_fixtures(
+            dsl.is_in('JP', z, x, y),
+            # https://www.openstreetmap.org/way/18576866
+            dsl.way(18576866, dsl.tile_diagonal(z, x, y), {
+                'bridge': 'yes', 'layer': '3', 'maxspeed': '80',
+                'bicycle': 'no',
+                'name': u'\u795e\u6238\u6de1\u8def\u9cf4\u9580\u81ea'
+                u'\u52d5\u8eca\u9053',
+                'source': 'openstreetmap.org', 'oneway': 'yes',
+                'surface': 'paved',
+                'bridge:name': u'\u660e\u77f3\u6d77\u5ce1\u5927\u6a4b',
+                'name:en': 'Kobe-Awaji-Naruto Expressway',
+                'motorcycle': 'designated', 'motorcar': 'designated',
+                'foot': 'no', 'lanes': '3', 'toll': 'yes', 'ref': 'E28',
+                'smoothness': 'excellent', 'highway': 'motorway',
+                'name:ja': u'\u795e\u6238\u6de1\u8def\u9cf4\u9580\u81ea'
+                u'\u52d5\u8eca\u9053',
+            }),
+            dsl.relation(1, {
+                'name:en': 'Kobe Awaji Naruto Expressway',
+                'name': u'\u795e\u6238\u6de1\u8def\u9cf4\u9580\u81ea'
+                u'\u52d5\u8eca\u9053', 'short_name': u'\u795e\u6238'
+                u'\u6de1\u8def\u9cf4\u9580\u9053', 'type': 'route',
+                'name:ja_kana': u'\u3053\u3046\u3079\u3042\u308f\u3058'
+                u'\u306a\u308b\u3068\u3058\u3069\u3046\u3057\u3083'
+                u'\u3069\u3046', 'name:ja_rm': u'Kobe-Awaji-Naruto Jid'
+                u'\u014dshad\u014d', 'source': 'openstreetmap.org',
+                'wikipedia': u'ja:\u795e\u6238\u6de1\u8def\u9cf4\u9580'
+                u'\u81ea\u52d5\u8eca\u9053', 'ref': 'E28', 'route': 'road',
+                'name:ja': u'\u795e\u6238\u6de1\u8def\u9cf4\u9580\u81ea'
+                u'\u52d5\u8eca\u9053',
+            }, ways=[18576866]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 18576866,
+                'shield_text': 'E28', 'network': 'JP:expressways'})
```

Comment:
Flagging that this network value is plural while others for Japan are singular, but consistent with `CN:expressways` so okay.

Reply:
Do you think we should make both `JP:expressways` and `CN:expressways` singular instead?